NNesta/trainee-app
# fix/Revamp-controllers

#### What does this PR do?
Make sure the whole AuthController is splitted into different categories (coach, trainee and users)
#### Description of Task to be completed?
- create another directory called userRoute
- Put all the routes that related to users like get-trainees, get-coaches,..etc
- Change in the server.js file so that we have the root called users in addition to only auth ( users/..., auth/...)
#### How should this be manually tested?
run `git clone https://github.com/NNesta/trainee-app.git`
run `cd trainee-app`
run `git checkout fix/revamping-controllers`
run `npm install`
run `npm run dev` and it should run successfully
test all the routes to see if you sucdessully fetch resources
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
None
